### PR TITLE
Gabbyk

### PR DESCRIPTION
**Pull request by gkokhabi**
Redux
styling for the card
card list
dashboard

### CODE REVIEW

**Comment by gkokhabi** [comment]
We love redux

**Review by TylerPohn — APPROVED** [review]
I see that you got the reducer working. Looks great.

**Review by rammarimuthu — APPROVED** [review]

**Comment by gkokhabi** [comment]
signup button on login page will need a different route route

File: client/components/ComplimentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            {!this.state.editMode ? <span>{tag}</span> : 
+            <form>
+              <select id='tag' name='tag'>
+                {options}
+              </select>
+            </form>
+            }
+
+            {!this.state.editMode ? <button onClick={this.handleEdit}>Edit Item</button> : 
+              <button onClick={() => this.handleSubmit()}>Submit</button>
```
Comment (gkokhabi) [inline client/components/ComplimentCard.jsx:54]: for the patch request

**Pull request merged**